### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by SolomonAfutu2003**
This is the final version of my portfolio project called "Desktop".
In the version, I
. Added new CSS styles.
. Added a media query.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,40 +188,75 @@ <h3>LET'S CONNECT</h3>
...
           </div>
           <div class="Lang-box">
             <div class="js-img-box">
-              <img src="img/Ellipse 1.png" alt="" />
               <ul>
-                <li>Javascript</li>
+                <li><img src="img/Ellipse 1.png" alt="" /></li>
+                <li class="L1">Javascript</li>
               </ul>
             </div>
```
Comment (abdulhamiid) [inline index.html:201]: Your design
image

Figma
image


According to [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md), we should replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Please kindly change the background color of the items to match the template on Figma 👍

File: index.html [hunk 2]
```diff
@@ -183,40 +188,75 @@ <h3>LET'S CONNECT</h3>
...
+        />
+        <textarea
+          name="message"
+          class="textarea"
+          placeholder="Enter your message here..."
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+        ></textarea>
```
Comment (abdulhamiid) [inline index.html:257]: Figma (input fields and the text area have the same width)
image

Your design (input fields have smaller width compared to the textarea)
image

Please adjust this section to ensure that `input` fields and the `textarea` have equal `width` as given in the Figma design. 🤗

File: README.md [hunk 3]
```diff
@@ -120,12 +120,18 @@ To run tests, run the following command:
...
 - GitHub: [@githubhandle](https://github.com/SolomonTheCoder)
 - Twitter: [@twitterhandle](https://twitter.com/SolomonAfutu)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/solomon-afutu-164b95255/)
 
+👤 **Binod Bhusal**
+
+- GitHub: [@githubhandle](https://github.com/binodbhusal)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (abdulhamiid) [inline README.md]: Please update this section's broken links to make your readme file professional. `Twitter` and `LinkedIn` URL. 👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @SolomonTheCoder 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @SolomonTheCoder 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**